driver/razerkraken_driver.c

```diff
@@ -215,6 +215,10 @@ static ssize_t razer_attr_read_device_type(struct device *dev, struct device_att
         device_type = "Razer Kraken 7.1 V2\n"; // Kylie
         break;
 
+    case USB_DEVICE_ID_RAZER_KRAKEN_TOURNAMENT:
```

Comment:
Change the defines to `USB_DEVICE_ID_RAZER_KRAKEN_TE` or `USB_DEVICE_ID_RAZER_KRAKEN_TOURNAMENT_EDITION` (shorter one is probably better, check if _TE or _TOURNAMENT_EDITION is used, I think the former)